src/Avalonia.Controls/Primitives/PopupRoot.cs

```diff
@@ -50,6 +56,7 @@ public PopupRoot(TopLevel parent, IPopupImpl impl, IAvaloniaDependencyResolver?
             : base(impl, dependencyResolver)
         {
             ParentTopLevel = parent;
+            impl.SetWindowManagerAddShadowHint(WindowManagerAddShadowHint);
```

Comment:
This isn't ideal but the default value of `false` for this property appears to be different to the default value implemented in the win32 backend (true), so this is needed to communicate the actual default. I could change the backend but didn't really want to mess with that here quite honestly.